src/Components/Server/src/Circuits/RemoteJSRuntime.cs

```diff
@@ -24,8 +24,8 @@ internal class RemoteJSRuntime : JSRuntime
         private readonly long _maximumIncomingBytes;
         private int _byteArraysToBeRevivedTotalBytes;
 
-        internal int RemoteJSDataStreamNextInstanceId;
-        internal readonly Dictionary<long, RemoteJSDataStream> RemoteJSDataStreamInstances = new();
+        internal int JSDataStreamNextInstanceId;
+        internal readonly Dictionary<long, RemoteJSDataStream> JSDataStreamInstances = new();
```

Comment:
Doesn't seem like these names need to change - they were pretty descriptive already